packages/perennial/package.json

```diff
@@ -43,7 +43,7 @@
     "@equilibria/emptyset-batcher": "0.1.0",
     "@equilibria/perennial-oracle": "1.2.0",
     "@equilibria/root": "0.2.0",
-    "@openzeppelin/contracts": "4.8.0"
+    "@openzeppelin/contracts": "4.6.0"
```

Comment:
is it ok to commit this?

Reply:
Yeah this is actually what we want, since we have deployed the core contracts with 4.6.0 already, I shouldn't have updated this package to 4.8.0 until we actually needed to